### DISCUSSION TITLE
Cardinal points - change letters

### DISCUSSION THREAD

**Discussion by fzs1933** [Stellarium/stellarium#2046] [Q&A] [answered]
Hi,

Me again...
Is it possible to change the letters of cardinal points and show only the four main directions?
(N -> É, E -> K, S -> D, W -> Ny)
Thanks
Zsolt

**Comment by gzotti** [comment, marked as answer]
The first point is handled by translation. I.e., select your language. If it is not translated, help translating to your language!
The second point is handled by checking one or more checkboxes: F4, tab "Markings", and select Cardinal Points with the two adjacent checkboxes for refinement.

**Reply by fzs1933** [reply]
Thanks, but 0.21.0 contains only one checkbox... :) 0.21.2 is already good.
How can I help with further translation?
![cardinals](https://user-images.githubusercontent.com/81897104/142424254-58f1ee64-8fb3-4537-b198-ccc5c6bb525e.png)
s 
Zsolt

**Reply by alex-w** [reply]
Please see https://www.transifex.com/stellarium/stellarium/dashboard/

**Reply by fzs1933** [reply]
OK, I send a join request
Thanks
Zsolt
